### DISCUSSION TITLE
CSS is not being applied to my html pages

### DISCUSSION THREAD

**Discussion by MorisetteA** [community/community#22832] [Other Feature Feedback, Questions, & Ideas]
<p>I have recently just finished coding my portfolio on Brackets (I’m not sure what version, I believe it is the most recent one). I have three folders in my repository and they are not linking to my HTML files.</p>
<p>Here is the link to my site: <a href="https://morisettea.github.io/portfolio/" rel="nofollow noopener">https://morisettea.github.io/portfolio/</a><br>
Here is my repository: <a href="https://github.com/MorisetteA/portfolio" rel="nofollow noopener">https://github.com/MorisetteA/portfolio</a></p>
<p>Everything works when I use the live preview on brackets and as well as clicking the file to display on a browser.</p>
<p>My pathway in my git repository and my computer is:<br>
_css (folder)<br>
_fonts(folder)<br>
_images(folder)<br>
index.html<br>
About.html<br>
(as well as a couple of more HTML files)</p>
<p>This is how I called my files.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/12dd8459a11823783cc46db404c6cc9dabc5b62a.jpeg" data-download-href="/uploads/short-url/2GTbVISGLmaHxBCMz3pPo6uZl4C.jpeg?dl=1" title="gitindex" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/50850277/181073285-e78128f9-b8d3-4aa2-afe3-044460661a82.jpeg" alt="gitindex" data-base62-sha1="2GTbVISGLmaHxBCMz3pPo6uZl4C" width="690" height="237" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/12dd8459a11823783cc46db404c6cc9dabc5b62a_2_690x237.jpeg, https://aws1.discourse-cdn.com/github/optimized/2X/1/12dd8459a11823783cc46db404c6cc9dabc5b62a_2_1035x355.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/1/12dd8459a11823783cc46db404c6cc9dabc5b62a_2_1380x474.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/12dd8459a11823783cc46db404c6cc9dabc5b62a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">gitindex</span><span class="informations">2618×903 86.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I’m not sure if I have my settings wrong in my git repository or something case-sensitive I am not aware of? Any help would be greatly appreciated, this is my first time doing this!</p>
<p>EDIT: I fixed it!!! so what the problem was is that I named my folders with an “_” like “_css” and that was the problem. Not sure why this would be an issue, I’ll have to research that.</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Glad you fixed it already! To give you some more info, and to leave some for anyone in the future that finds this, GitHub Pages uses Jekyll to build sites by default. Jekyll excludes files and directories that start with certain special characters like <code>_</code> unless you explicitly include them inside your <code>_config.yml</code> file. You can see a list of them here:</p>
<p><a href="https://docs.github.com/en/github/working-with-github-pages/about-github-pages-and-jekyll#configuring-jekyll-in-your-github-pages-site" class="onebox" target="_blank" rel="nofollow noopener">https://docs.github.com/en/github/working-with-github-pages/about-github-pages-and-jekyll#configuring-jekyll-in-your-github-pages-site</a></p>
<p>To prevent this in future you can either:</p>
<ul>
<li>Rename the file or directory</li>
<li>Explicitly include the file/directory in the <code>_config.yml</code> file</li>
<li>Disable Jekyll altogether if you’re not using it by creating an empty file called <code>.nojekyll</code> in the root of your Pages publishing source</li>
</ul>

**Comment by MorisetteA** [comment]
<p>Thank you so much for the response and the prevention tips!!! Your explanation really helped, I had not idea about Jekyll excluding filed with special characters.</p>

**Comment by paulbizsunny** [comment]
<p>I’m still facing same issues as well how may I come out of the challenge please</p>

**Comment by paulbizsunny** [comment]
<p>CC is not being applied to my html page</p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Hi <a class="mention" href="/paulbizsunny">@paulbizsunny</a> We are happy you are here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right directions.</p>
<aside class="quote no-group" data-username="paulbizsunny" data-post="5" data-topic="122136" data-full="true">
<div class="title">
<div class="quote-controls"></div>
 paulbizsunny:</div>
<blockquote>
<p>CC is not being applied to my html page</p>
</blockquote>
</aside>
<p>If your issue is different from the user’s issue and solution above in this topic, we encourage you to open a new topic with the detail of your source code or public repository that you would like help with.</p>

**Comment by paulbizsunny** [comment]
<p>No I’m satisfied, I don’t have new topic you know I’m newly for the field I’m still learning thank you</p>

**Comment by Ericnk1** [comment]
<p>Hi, Could you please assist me with with similar issue? The css wasn’t applied to my page, I changed few things it worked but not correctly.<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <a href="https://ericnk1.github.io/Pet-ClinicUI/" target="_blank" rel="noopener nofollow ugc">ericnk1.github.io</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="" height="">

<h3><a href="https://ericnk1.github.io/Pet-ClinicUI/" target="_blank" rel="noopener nofollow ugc">Pet Clinic</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/ericnk1">@Ericnk1</a>, welcome to the GitHub Support Community!</p>
<p>I’m seeing CSS being applied to that page correctly. Did you manage to figure this out or are you still having issues somewhere?</p>

**Comment by Ericnk1** [comment]
<p>Hi, when I use Edge, the css is well applied just like on my localhost. But with Chrome not very well. Also the Script file, JQuery, and other files from the node_modules folder can’t be accessed even though I added the node_modules in there.</p>
<p>Thank you very much for your assistance.</p>

**Comment by Ericnk1** [comment]
<p>It’s working now.<br>
Thank you very much!</p>

**Comment by divya-mohan0209** [comment]
<p>Hello folks, good day!</p>
<p>I’m not having CSS rendered for the pages under this particular URL - <a href="https://divya-mohan0209.github.io/docusaurus-current-theme/rucio/" rel="noopener nofollow ugc">https://divya-mohan0209.github.io/docusaurus-current-theme/rucio/</a></p>
<p>The CSS, however, along with the imaging is being rendered perfectly for pages one level up - <a href="https://divya-mohan0209.github.io/docusaurus-current-theme/" rel="noopener nofollow ugc">https://divya-mohan0209.github.io/docusaurus-current-theme/</a></p>
<p>A bit more about my project:</p>
<ul>
<li>I am using Sphinx to convert Python RST files into HTML</li>
<li>The HTML files are built into the rucio directory and are published on to GH-pages using docusaurus.</li>
</ul>
<p>Docusaurus is being used since we anticipate the requirement of MDX support.</p>
<p>I’ve included a .nojekyll file per the suggestions here in the rucio directory, since I’m aware Jekyll excludes stuff after _ in a file.</p>
<p>Since I do not have much control the files generated by sphinx/Docusaurus, i.e. _static, _css etc. could you please suggest if I’m going wrong somewhere?</p>
<p>Regards,<br>
Divya</p>

**Comment by divya-mohan0209** [comment]
<p>Just to add on, it’s being rendered seamlessly on my local machine so I genuinely believe it has something to do with the jekyll bit.</p>
